spiceaidocs/docs/components/data-connectors/odbc.md

```diff
@@ -4,6 +4,21 @@ sidebar_label: 'ODBC Data Connector'
 description: 'ODBC Data Connector Documentation'
 ---
 
+## Setup
+
+:::warning
+
+ODBC support is not included in the released binaries. To use ODBC with Spice, you need to [checkout and compile the code](https://github.com/spiceai/spiceai/blob/trunk/CONTRIBUTING.md#building) with the `--features odbc` flag (`cargo build --release --features odbc`).
+
+:::
+
+An ODBC connection requires a compatible ODBC driver and valid driver configuration. ODBC drivers are available from their respective vendors. Here are a few examples:
```

Comment:
It requires two components, a driver manager (i.e. `unixodbc`) and a driver to connect to the database. On Windows the driver manager is built-in, but Linux and macOS require `unixodbc` to be installed